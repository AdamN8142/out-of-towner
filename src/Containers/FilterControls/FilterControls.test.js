describe('FilterControls', () => {
  it('should match snapshot', () => {
    let wrapper
    expect(wrapper).toMatchSnapshot()
  })
})