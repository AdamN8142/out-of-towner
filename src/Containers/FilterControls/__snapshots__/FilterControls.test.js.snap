// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FilterControls should match snapshot 1`] = `undefined`;
